scene/3d/path_3d.h

```diff
@@ -72,14 +72,17 @@ class PathFollow3D : public Node3D {
 		ROTATION_ORIENTED
 	};
 
+	static Transform3D correct_posture(Transform3D p_transform, PathFollow3D::RotationMode p_rotation_mode);
+
 private:
 	Path3D *path = nullptr;
-	real_t prev_offset = 0.0; // Offset during the last _update_transform.
+	real_t prev_offset = 0.0; // Offset during the last _update_transform. //not used, redundant?
```

Comment:
Probably you can remove it, since you have made the sampling to be done by Curve.

Reply:
done